components/usage/pkg/apiv1/usage.go

```diff
@@ -136,6 +136,20 @@ func (s *UsageService) ReconcileUsage(ctx context.Context, req *v1.ReconcileUsag
 
 }
 
+func (s *UsageService) GetCostCenter(ctx context.Context, in *v1.GetCostCenterRequest) (*v1.GetCostCenterResponse, error) {
+	result, err := db.GetCostCenter(ctx, s.conn, db.AttributionID(in.AttributionId))
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Failed to get cost center: %s", err)
```

Comment:
Can we do better here? Can we identify if it's an error because it doesn't exist?